examples/recipes/llama/distill_llama32_3b-1b.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
```

Comment:
I'd strongly recommend keeping the examples to be as python-based as possible with minimal config dependencies. I am planning to rework the llama recipe example as it was added initially to demonstrate config overrides, but i think it ends up detracting from seeing the main APIs at work.